src/subdomains/voting/api/voting.controller.ts

```diff
@@ -38,6 +38,12 @@ export class VotingController {
     return this.votingService.result;
   }
 
+  @Get('user-result')
+  @ApiOkResponse({ type: CfpResultDto, isArray: true })
+  async getCurrentUserResult(): Promise<CfpUserInfo[]> {
+    return this.votingService.getVoteDetails();
+  }
```

Comment:
Naming could be more consistent, is it `user result`, `CFP user info` or `vote details`? 😉
Maybe `result details` or `user votes` or similar?